python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -988,60 +853,54 @@ interface {{ ethernet_interface.name }}
 {%         endif %}
-{%         if ethernet_interface.sflow.egress.enable is arista.avd.defined(true) %}
-   sflow egress enable
-{%         elif ethernet_interface.sflow.egress.enable is arista.avd.defined(false) %}
-   no sflow egress enable
+   {{ backup_link_cli }}
+{%         if ethernet_interface.switchport.backup.preemption_delay is arista.avd.defined %}
+   switchport backup preemption-delay {{ ethernet_interface.switchport.backup.preemption_delay }}
 {%         endif %}
-{%         if ethernet_interface.sflow.egress.unmodified_enable is arista.avd.defined(true) %}
-   sflow egress unmodified enable
-{%         elif ethernet_interface.sflow.egress.unmodified_enable is arista.avd.defined(false) %}
-   no sflow egress unmodified enable
+{%         if ethernet_interface.switchport.backup.mac_move_burst is arista.avd.defined %}
+   switchport backup mac-move-burst {{ ethernet_interface.switchport.backup.mac_move_burst }}
+{%         endif %}
+{%         if ethernet_interface.switchport.backup.mac_move_burst_interval is arista.avd.defined %}
+   switchport backup mac-move-burst-interval {{ ethernet_interface.switchport.backup.mac_move_burst_interval }}
+{%         endif %}
+{%         if ethernet_interface.switchport.backup.initial_mac_move_delay is arista.avd.defined %}
+   switchport backup initial-mac-move-delay {{ ethernet_interface.switchport.backup.initial_mac_move_delay }}
+{%         endif %}
+{%         if ethernet_interface.switchport.backup.dest_macaddr is arista.avd.defined %}
+   switchport backup dest-macaddr {{ ethernet_interface.switchport.backup.dest_macaddr }}
 {%         endif %}
 {%     endif %}
-{%     if ethernet_interface.vmtracer is arista.avd.defined(true) %}
-   vmtracer vmware-esx
-{%     endif %}
-{%     if ethernet_interface.transceiver.media.override is arista.avd.defined %}
-   transceiver media override {{ ethernet_interface.transceiver.media.override }}
-{%     endif %}
-{%     if ethernet_interface.transceiver.frequency is arista.avd.defined %}
-{%         set frequency_cli = 'transceiver frequency ' ~ "%.3f" | format(ethernet_interface.transceiver.frequency | float) %}
-{%         if ethernet_interface.transceiver.frequency_unit is arista.avd.defined %}
-{%             set frequency_cli = frequency_cli ~ ' ' ~ ethernet_interface.transceiver.frequency_unit %}
+{%     if ethernet_interface.sync_e.enable is arista.avd.defined(true) %}
+   sync-e
+{%         if ethernet_interface.sync_e.priority is arista.avd.defined %}
+      priority {{ ethernet_interface.sync_e.priority }}
 {%         endif %}
-   {{ frequency_cli }}
 {%     endif %}
 {%     for link_tracking_group in ethernet_interface.link_tracking_groups | arista.avd.natural_sort %}
 {%         if link_tracking_group.name is arista.avd.defined and link_tracking_group.direction is arista.avd.defined %}
    link tracking group {{ link_tracking_group.name }} {{ link_tracking_group.direction }}
```

Comment:
seems to be a bug, i am not able to configure multiple group, can you check again

Reply:
We can configure multiple 

```
   sync-e
      priority 22
   link tracking group qwe upstream
   link tracking group pppopo upstream
   vmtracer vmware-esx
```

Reply:
But if we configure `downstream` when `upstream` is already configured we get a warning on cli

```
s1-spine1(config-if-Et5)#link tracking group dummy downstream 
! Interface already present in group qwe as upstream interface
! Interface already present in group pppopo as upstream interface
! Cannot configure interface as requested
```

This would require schema change 

Current schema:

```
link_tracking_groups:
          type: list
          primary_key: name
          items:
            type: dict
            keys:
              name:
                type: str
                description: Group name.
              direction:
                type: str
                valid_values: ["upstream", "downstream"]
```
Wee need to move `direction` out of list items. @ClausHolbechArista

Reply:
Not now. We need to finish the reordering. You can submit an issue with this for later.

Reply:
Sure, I will raise an issue.

Reply:
Raised an issue 
https://github.com/aristanetworks/avd/issues/4584